services/galley/test/integration/Federation.hs

```diff
@@ -0,0 +1,374 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# OPTIONS_GHC -Wno-unused-matches #-}
+
+module Federation where
```

Comment:
this should also not have gone here.  let's move the background-worker suite first and see how painful it is, though.